turbopy/core.py

```diff
@@ -724,6 +724,89 @@ def interpval(yvec):
             return interpval
 
 
+    def set_cell_volumes(self, coordinate_system):
+        """
+        This function sets some volume-related elements that are useful for
+        finite difference solvers in curvilinear coordinates.
+
+        This function set the following volume-realted quantities:
+
+        old_cell_volume(cell.size) - These are the volume elements at the beginning
+                                     of the time step.
+        cell_volume(cell.size)     - These are the volume elements at the end
+                                     of the time step.
+        interface_volume(edge.size) - Average of adjacent cell volumes
+        inverse_cell_volume(cell.size)   -  1/cell_volume
+        """
+        old_grid = self.cell_edges  # Old grid at beginning of time step
+        new_grid = self.cell_edges  # New grid at end of time step
+        fourthirds = 4/0/3.0
+        if coordinate_system.lower().strip() == 'cartesian':
+            self.old_cell_volume = old_grid[1:] - old_grid[0:-1]
+            self.cell_volume = new_grid[1:] - new_grid[0:-1]
+        elif coordinate_system.lower().strip() == 'cylindrical':
+            scrh = old_grid * old_grid
+            self.old_cell_volume = np.pi * (scrh[1:] - scrh[0:-1])
+            scrh = new_grid * new_grid
+            self.cell_volume = np.pi * (scrh[1:] - scrh[0:-1])
+        elif coordinate_system.lower().strip() == 'spherical':
+            scrh = old_grid * old_grid * old_grid
+            self.old_cell_volume = fourthirds*np.pi*(scrh[1:]- scrh[0:-1])
+            scrh = new_grid * new_grid * new_grid
+            self.cell_volume = fourthirds*np.pi*(scrh[1:]- scrh[0:-1])
+        else:
+            raise Exception('grid type not defined')
+        self.inverse_volume = 1./self.cell_volume
+
+
+    def set_interface_volumes(self, coordinate_system):
+        """
+        This function sets some volume-related elements that are useful for
```

Comment:
Like the above, please remove this docstring.

Reply:
The docstrings need to be in numpydoc format.